source/release/4.2.0/index.html.md

```diff
@@ -0,0 +1,710 @@
+
+- **Virt-v2v** - the tool that converts VMs from a foreign hypervisor to run on KVM - now supports **Debian/Ubuntu** based VMs, in addition to the supported RPM and Windows-based VMs. It is available for VDSM hosts running on RHEL 7.4, and from oVirt hosts versions 4.0 and above.
+
+- oVirt 4.2.0 uses **PostgresSQL 9.5** as its database, for improved performance.
+
+- Support is now restored for **Gluster ISO domains**, without the need for NFS which was previously disabled by Gluster. 
+
+- **Affinity labels** create a positive affinity between the hosts and VMs to which they are applied. The new Affinity Labels sub tab for clusters, hosts, and VMs in the Administration Portal provides a table view of labels associated with the currently selected entity, as well as the option to add, edit, and delete them. Additionally, existing labels can be added and/or removed from VMs and hosts in their respective new/edit popup dialogs.
+
+To learn about features introduced before 4.2.0, see the [release notes for previous versions](/documentation/#previous-release-notes).
+
+
+## Install / Upgrade from previous versions
+
+### Fedora / CentOS / RHEL
+
+
+## ALPHA RELEASE
+
+In order to install this Alplha Release you will need to enable pre-release repository.
+
+
+In order to install it on a clean system, you need to install
+
+
+`# yum install `[`http://resources.ovirt.org/pub/yum-repo/ovirt-release42-pre.rpm`](http://resources.ovirt.org/pub/yum-repo/ovirt-release42-pre.rpm)
+
+
+and then follow our
+[Installation Guide](http://www.ovirt.org/documentation/install-guide/Installation_Guide/)
+
+
+
+### oVirt Hosted Engine
+
+If you're going to install oVirt as Hosted Engine on a clean system please
+follow [Hosted_Engine_Howto#Fresh_Install](/documentation/how-to/hosted-engine/#fresh-install)
+guide or the corresponding section in
+[Self Hosted Engine Guide](/documentation/self-hosted/Self-Hosted_Engine_Guide/).
+
+If you're upgrading an existing Hosted Engine setup, please follow
+[Hosted_Engine_Howto#Upgrade_Hosted_Engine](/documentation/how-to/hosted-engine/#upgrade-hosted-engine)
+guide or the corresponding section within the
+[Upgrade Guide](/documentation/upgrade-guide/upgrade-guide/).
+
+### EPEL
+
+TL;DR Don't enable all of EPEL on oVirt machines.
+
+The ovirt-release package enables the epel repositories and includes several
+specific packages that are required from there. It also enables and uses
+the CentOS SIG repos, for other packages.
+
+If you want to use other packages from EPEL, you should make sure to
+use `includepkgs` and add only those you need avoiding to override
+packages from other repos.
+
+### Deprecated Functionality
```

Comment:
Suggest to move this section BELOW "enhancements"

Reply:
I'll try to change the tool for autogeneration setting the requested order.